packages/memory-storage/src/workers/worker-utils.ts

```diff
@@ -65,6 +65,11 @@ async function updateItems(message: WorkerUpdateEntriesMessage) {
     const dir = message.entityDirectory;
     await ensureDir(dir);
 
+    // Skip writing files to the disk if the client has the option set to false
+    if (!message.writeFilesToDisk) {
```

Comment:
maybe lets put this above the `ensureDir` call? i think the goal is to disable file system access in general, not just "dont write files, but folders are fine" :]